buildbot/master.cfg

```diff
@@ -578,6 +578,45 @@ def make_android_package(mode="normal"):
 
     return f
 
+def make_dolphin_freebsd_build(mode="normal"):
+    f = BuildFactory()
+
+    mode = mode.split(",")
+
+    nogui = "nogui" in mode
+
+    f.addStep(GitNoBranch(repourl="https://github.com/dolphin-emu/dolphin.git",
+                          progress=True, mode="incremental"))
+
+    f.addStep(ShellCommand(command=["mkdir", "-p", "build"],
+                           logEnviron=False,
+                           description="mkbuilddir",
+                           descriptionDone="mkbuilddir"))
+
+    cmake_cmd = ["cmake", ".."]
+    if nogui:
+        cmake_cmd.append("-DDISABLE_WX=ON")
+    f.addStep(ShellCommand(command=cmake_cmd,
+                           workdir="build/build",
+                           description="configuring",
+                           descriptionDone="configure",
+                           haltOnFailure=True,
+                           env={ "WX_CONFIG": "wxgtk2u-3.0-config" }))
```

Comment:
Looks like that's not possible, since the [FindwxWidgets.cmake](http://www.cmake.org/gitweb?p=cmake.git;a=blob;f=Modules/FindwxWidgets.cmake;h=dec0ddf20ca3593f119fd8e6e71fdfe208b7aeb8;hb=b3de2a8e6451c9138948c23a974740823ade8263#l743) program expects the program to be called `wx-config` and will give up otherwise. The only way to pass a hint is within the environment, which is either this line right here, or the way I start the builder. I don't think you can modify the environment of a currently running CMake script.